### ISSUE TITLE
ListOfConstraintIndices not exposed?

### ISSUE THREAD

**Issue by matbesancon** [scipopt/SCIP.jl#199]
```
o = SCIP.Optimizer

julia> MOI.get(o, MOI.ListOfConstraintIndices{MOI.ScalarAffineFunction{Float64}, MOI.LessThan{Float64}}())
ERROR: ArgumentError: ModelLike of type SCIP.Optimizer does not support accessing the attribute MathOptInterface.ListOfConstraintIndices{MathOptInterface.ScalarAffineFunction{Float64}, MathOptInterface.LessThan{Float64}}()
Stacktrace:
 [1] get_fallback(::SCIP.Optimizer, ::MathOptInterface.ListOfConstraintIndices{MathOptInterface.ScalarAffineFunction{Float64}, MathOptInterface.LessThan{Float64}})
   @ MathOptInterface ~/.julia/packages/MathOptInterface/YDdD3/src/attributes.jl:296
 [2] get(::SCIP.Optimizer, ::MathOptInterface.ListOfConstraintIndices{MathOptInterface.ScalarAffineFunction{Float64}, MathOptInterface.LessThan{Float64}})
   @ MathOptInterface ~/.julia/packages/MathOptInterface/YDdD3/src/attributes.jl:293
 [3] top-level scope
   @ REPL[12]:1

julia> MOI.get(o, MOI.ListOfConstraintIndices{MOI.SingleVariable, MOI.LessThan{Float64}}())
ERROR: ArgumentError: ModelLike of type SCIP.Optimizer does not support accessing the attribute MathOptInterface.ListOfConstraintIndices{MathOptInterface.SingleVariable, MathOptInterface.LessThan{Float64}}()
```

**Comment by matbesancon** [comment]
I'm not sure how things even worked without this implemented

**Comment by matbesancon** [comment]
closed by #203
